lighthouse-core/audits/estimated-input-latency.js

```diff
@@ -56,7 +56,8 @@ class EstimatedInputLatency extends Audit {
       const latencyPercentiles = TracingProcessor.getRiskToResponsiveness(model, trace, startTime);
 
       const ninetieth = latencyPercentiles.find(result => result.percentile === 0.9);
-      const rawValue = ninetieth.time.toFixed(1) + 'ms';
+      const rawValue = Number(ninetieth.time.toFixed(1));
```

Comment:
SG. I was mostly refering to the fact you have `rawValue` and `displayValue` but then you set `AuditResult.rawValue` to `displayValue` and not `rawValue` :)